core/modules/flowgen.cc

```diff
@@ -200,6 +200,13 @@ pb_error_t FlowGen::ProcessArguments(const bess::pb::FlowGenArg &arg) {
     quick_rampup_ = 1;
   }
 
+  if (arg.burst() <= bess::PacketBatch::kMaxBurst) {
```

Comment:
Could you separate `set_burst()` from this function, like other modules (such as `Source`, `PortInc`, etc.)? By having a separate function one can adjust burstiness of modules without doing `bess.pause_all()`